package.json

```diff
@@ -86,7 +86,7 @@
             {
                 "command": "jupyter.runcurrentcell",
                 "key": "ctrl+enter",
-                "when": "editorTextFocus && !editorHasSelection && jupyter.hascodecells && !notebookEditorFocused"
+                "when": "editorTextFocus && !editorHasSelection && jupyter.hascodecells && !notebookEditorFocused && !jupyter.hasnativecells"
```

Comment:
You have has here, it's actually havenativecells. Annoying I know, wish we could actually have safe typing on these :/